### DISCUSSION TITLE
i3wm doesn't capture key bindings when systray icons are clicked

### DISCUSSION THREAD

**Discussion by afro-coder** [i3/i3#4595] [Q&A] [answered]
Hi,

So I have binded Flameshot for screenshots
```
bindsym --release $mod+Shift+s exec flameshot GUI
```
But if I click on any system try icon like Pasystray 
I cant' get the flameshot GUI to come up, no errors in journald too.

It works flawlessly otherwise.

**Comment by Airblader** [comment, marked as answer]
You mean clicking on the tray icon and a menu being open? This is expected, because such menus generally grab the keyboard/pointer and prevent i3 from receiving the key events.

**Reply by afro-coder** [reply]
Ah okay, is there any sort of workaround, because the questions I had regarding pipewire and pasystray required me to click a screenshot of that tray.

**Reply by Airblader** [reply]
The only way to take a screenshot of such a context menu that I'm aware of is launching the command with a sleep before opening the menu, and having specified which area to take a screenshot of.

**Reply by afro-coder** [reply]
Yeah that is something that I can work on thanks.
